src/services/ColorThemePlugin.js

```diff
@@ -19,8 +19,16 @@ export const ColorThemePlugin = {
       Object.assign(defaultOptions.themes, options.themes)
     }
 
+    let themeCache = Object.assign({}, defaultOptions.themes)
+
     Vue.prototype.$themes = defaultOptions.themes
 
+    Vue.prototype.$themes.restoreThemes = () => {
```

Comment:
That shouldn't be a public api. Please just implement in component.

Reply:
Fixed in PR in vuestic-admin repository referenced above.

Reply:
@ssemenkoff I'm not following? Do I close this PR then?

"vue-epic-bus": "^0.1.3",
This change seems useful.

Reply:
@asvae You can merge it now.